compiler/src/iree/compiler/Dialect/Stream/Transforms/test/elide_timepoints.mlir

```diff
@@ -0,0 +1,84 @@
+// RUN: iree-opt --split-input-file --iree-stream-elide-timepoints %s | FileCheck %s
+
+// Tests that an immediate timepoint passed along a call edge is propagated.
+
+// DO NOT SUBMIT
+// function call/return
```

Comment:
you have a few `DO NOT SUBMIT` tests in this file

Reply:
crap, forgot that I was procrastinating on these tests :P good catch!